libs/payments/paypal/src/lib/paypalCustomer/paypalCustomer.factories.ts

```diff
@@ -0,0 +1,53 @@
+import { faker } from '@faker-js/faker';
+import {
+  CreatePaypalCustomer,
+  ResultPaypalCustomer,
+  UpdatePaypalCustomer,
+} from './paypalCustomer.types';
+
+export const ResultPaypalCustomerFactory = (
+  override?: Partial<ResultPaypalCustomer>
+): ResultPaypalCustomer => ({
+  ...override,
+  uid: faker.string.hexadecimal({
+    length: 32,
+    prefix: '',
+    casing: 'lower',
+  }),
```

Comment:
should we have a helper for this? Looks like we use this in about 5&times; in this PR.

Reply:
It's a method call to faker, so not much to put into a helper. Since these are already helpers (factories) I believe that's a bit too much abstraction - a wrapper (factory) of a wrapper (util) of a wrapper (faker).

Additionally, the code currently falls in-line with the style guide for the repo for factories, which should be explicit and fairly verbose.